apps/admin-gui/src/app/shared/components/dialogs/edit-application-form-item-dialog/edit-application-form-item-dialog.component.html

```diff
@@ -217,6 +217,13 @@ <h1 mat-dialog-title>{{'DIALOGS.APPLICATION_FORM_EDIT_ITEM.TITLE' | translate}}
               <mat-form-field>
                 <input matInput [(ngModel)]="option[1]">
               </mat-form-field>
+              <button
+                mat-icon-button
+                color="warn"
+                matTooltip="{{'DIALOGS.APPLICATION_FORM_EDIT_ITEM.REMOVE_OPTION' | translate}}"
+                (click)="removeOption(option, lang)">
+                <mat-icon>delete</mat-icon>
```

Comment:
```suggestion
                <mat-icon>remove_circle_outline</mat-icon>
```
```suggestion
                <mat-icon>clear</mat-icon>
```
some of that would be nicer according to me

Reply:
I agree, it looks better. I chose the `clear` icon.